deployment/helm/skaha/skaha-config/launch-desktop.yaml

```diff
@@ -89,6 +89,8 @@ spec:
           protocol: TCP
           name: novnc-port
         volumeMounts:
+        - mountPath: /dev/shm
+          name: dshm
         - mountPath: "/etc/passwd"
           name: etc-passwd
           subPath: passwd
```

Comment:
Desktop won't need extra shared memory (not user facing) so we should take it out of here.